client/my-sites/purchases/subscriptions/subscriptions-content.tsx

```diff
@@ -0,0 +1,81 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { useSelector } from 'react-redux';
+import { useTranslate } from 'i18n-calypso';
+
+/**
+ * Internal Dependencies
+ */
+import PurchasesSite from 'me/purchases/purchases-site/index.jsx';
+import {
+	getSitePurchases,
```

Comment:
It's interesting to me that we use `getSitePurchases` but not `hasLoadedSitePurchasesFromServer` or `isFetchingSitePurchases`. I haven't tracked down the difference between the two requests, but notice the site requests finish after the user requests. I wonder if this could be causing the blank slate/loading issue Filippo was seeing in his review.

Reply:
I don't fully understand this comment. We are using all three (the difference is not clear to me either) in `SubscriptionsContent`. FWIW this logic was copied from the existing `PurchasesList` component, although it's been converted to a functional component so it's possible that I got something wrong.

Reply:
I may have found a case where the placeholder might not be displayed even though we're still loading and I've tried to fix it in my most recent commit.